test/libyul/yulOptimizerTests/expressionSimplifier/sub_eq.yul

```diff
@@ -0,0 +1,16 @@
+{
+    let a := calldataload(0)
+    let b := calldataload(0x20)
+    let x := sub(a, b)
+    if gt(x, 0) {
```

Comment:
```suggestion
    if iszero(x) {
```
Shouldn't it be this?

Reply:
Well ``gt(x,0)`` will first become ``iszero(iszero(x))`` (explaining the additional ``iszero``) in the expectations. Not sure there's a good reason to complicate it like that, though, so I took the liberty to just change it.